service/templates/task.html

```diff
@@ -239,6 +239,10 @@ <h1 class="one-liner" id="task-title">
                   <td class="icon"><i class="fa fa-ban"></i></td>
                   <td>Blacklisted</td>
                 </tr>
+                <tr ng-show="task.display_task">
+                  <td class="icon"><i class="fa fa-link"></i></td>
+                  <td><a href="/task/[[task.display_task]]">Link to parent display task</a></td>
```

Comment:
I don't think links should have the word "link" in them. Maybe just "Parent" or "Parent display task"

Reply:
changed